Packs/Carbon_Black_Enterprise_Response/Integrations/CarbonBlackResponseV2/CarbonBlackResponseV2.py

```diff
@@ -855,8 +855,14 @@ def fetch_incidents(client: Client, max_results: int, last_run: dict, first_fetc
                 continue
 
         alert_id = alert.get('unique_id', '')
-        alert_name = alert.get('process_name', '')
-        incident_name = f'{INTEGRATION_NAME}: {alert_id} {alert_name}'
+        process_name = alert.get('process_name','')
+        if alert.get('feed_name', '') == 'My Watchlists':
+            alert['description'] = alert.get('watchlist_name', '')
+        alert_name = alert.get('description', '')
```

Comment:
custom watchlist names are stored in "watchlist_name" and feeds coming from other sources store their names in the "description" field